packages/simcore-sdk/src/simcore_sdk/node_ports_v2/port_utils.py

```diff
@@ -168,6 +169,7 @@ async def pull_file_from_store(
     key: str,
     fileToKeyMap: Optional[dict[str, str]],
     value: FileLink,
+    io_log_redirect_cb: Optional[LogRedirectCB],
```

Comment:
why did not you choose to have ``None`` by default?

Reply:
hmm yes. so I tend to put the default None to the external interface, not the internals of the library. I think defaults are usually a bit dangerous as people tend to not set them explicitly.